front/src/common/SelectorSNCF.tsx

```diff
@@ -35,53 +49,82 @@ export default function SelectorSNCF<T extends string | null, K extends string>(
     onItemRemoved,
     translationFile,
     translationList = '',
+    extraColumn,
   } = props;
   const { t } = useTranslation(translationFile);
 
   return (
-    <div className={`${mainClass}`}>
+    <div className={`${mainClass}${extraColumn ? '-withExtraColumn' : ''}`}>
```

Comment:
I find this clearer, what do you think ?
```suggestion
    <div className={extraColumn ? `${mainClass}-withExtraColumn` : `${mainClass}`}>
```

Reply:
Yes totally ! Done.

Reply:
Thanks :pray: